test/fakes/fake-xrp-protobufs.ts

```diff
@@ -0,0 +1,193 @@
+import {
```

Comment:
This is so great.

Reply:
I think a refactor I might do is to have all the protobuf conversion tests reference objects from this file

Reply:
Feel free but we can also just do it over time. You could also add this as a bootcamp in Asana if you think it's a good candidate.

Reply:
That's a good idea - I'll do this.